plugins/org.jboss.tools.openshift.client/META-INF/MANIFEST.MF

```diff
@@ -17,19 +17,57 @@ Bundle-ClassPath: lib/openshift-restclient-java-1.0.0-SNAPSHOT.jar,
  lib/commons-lang-2.6.jar,
  lib/commons-io-2.1.jar,
  .
-Export-Package: com.openshift.internal.restclient.http,
- com.openshift.restclient,
- com.openshift.restclient.authorization,
- com.openshift.restclient.capability,
- com.openshift.restclient.capability.resources,
- com.openshift.restclient.capability.server,
- com.openshift.restclient.http,
- com.openshift.restclient.images,
- com.openshift.restclient.model,
- com.openshift.restclient.model.build,
- com.openshift.restclient.model.route,
- com.openshift.restclient.model.template,
- com.openshift.restclient.utils
+Export-Package: com.openshift.internal.restclient,
+ com.openshift.internal.restclient.http,
+ com.openshift.restclient;
+  x-friends:="org.jboss.tools.openshift.core,
```

Comment:
Why add the x-friends back in? I removed those a few weeks ago to make sure any other plugin can use the client plugin. Please remove the x-friends.